crates/ide/src/moniker.rs

```diff
@@ -256,8 +256,8 @@ pub(crate) fn def_to_moniker(
             let (name, repo, version) = match krate.origin(db) {
                 CrateOrigin::CratesIo { repo, name } => (
                     name.unwrap_or(krate.display_name(db)?.canonical_name().to_string()),
-                    repo?,
-                    krate.version(db)?,
+                    repo.unwrap_or_default(),
+                    krate.version(db).unwrap_or_default(),
```

Comment:
An alternative would be to use `unwrap_or_else(|| "file:///crate/path".into())` and `unwrap_or_else(|| "v0.0.0".into())` or so.

Let me know if that's somehow preferable (though that would make the symbols for local crates non-deterministic, which would be somewhat unfortunate).